checklist.html

```diff
@@ -1,242 +1,1029 @@
+          Terms like "click here" and "read more" do not provide any context. Some people navigate using a list of all buttons or links on a page or view. When using this mode, the terms indicate what will happen if navigated to or activated.
+        </p>
+      </div>
+    </details>
+    <details id="section-content-03">
+      <summary>
+        Use left-aligned text for left-to-right languages, and right-aligned text for right-to-left languages.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a>
+        </p>
+        <p class="checklist-description">
+          Centered-aligned or justified text is difficult to read.
+        </p>
+      </div>
+    </details>
+  </div>
+</section>
+
+<section class="article-section resources-section checklist-section" id="section-global-code" aria-labelledby="label-global-code">
+  <h2 id="label-global-code">Global code</h2>
+  <p>
+    Global code is code that affects your entire website.
+  </p>
+  <div class="checklist">
+    <details id="section-global-code-01">
+      <summary>
+        Validate the HTML you generate.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-parses.html">4.1.1 Parsing</a>
+        </p>
+        <p class="checklist-description">
+          <a href="https://validator.w3.org/nu/">Valid HTML</a> helps give a consistent experience across all browsers and assistive technology.
+        </p>
+      </div>
+    </details>
+    <details id="section-global-code-02">
+      <summary>
+        Use a <code>lang</code> attribute on the <code>html</code> element.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-doc-lang-id.html">3.1.1 Language of Page</a>
+        </p>
+        <p class="checklist-description">
+          This helps screen readers to <a href="https://github.com/FreedomScientific/VFO-standards-support/issues/188">pronounce content correctly</a>.
+        </p>
+      </div>
+    </details>
+    <details id="section-global-code-03">
+      <summary>
+        Provide a unique <code>title</code> for each page or view.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-title.html">2.4.2 Page Titled</a>
+        </p>
+        <p class="checklist-description">
+          The <code>title</code> element, contained in the document's <code>head</code> element, is often the first piece of information announced by assistive technology. This helps tell people what screen or view they are going to start navigating.
```

Comment:
We should settle on a term and add it to the writing styleguide.

I've been using "page/view" to specifically name both SPA and non-SPA use cases, as I think it'll preempt potential arguments about the whole "SPAS aren't documents!" crowd.

Reply:
yeh.  but they're only right in that it's not document*s*.  SPAs are still very much still a web document... it just keeps getting rewritten instead of allowing navigation between different documents.

It's really just a silly thing to argue about imo. :)